api/tasks.py

```diff
@@ -11,3 +14,22 @@ def create_delivery(order):
     else:
         delivery.print_pdf()
         return {'success': 'all good'}
+
+
+@app.task
+def update_customer_if_is_partner(email):
+    if check_customer_is_partner(email):
+        # here will come the code to update the Wordpress user role.
+        pass
+
+
+def check_customer_is_partner(email):
```

Comment:
isn't the convention in Python to prefix private methods with `_`? IMO it's critical to draw a line between public and private methods.